trin-state/src/utils.rs

```diff
@@ -75,84 +53,55 @@ mod test {
     use super::*;
     use test_log::test;
 
-    // 2 ** 256 - 1
-    const MOD_SUB_ONE: &str =
-        "115792089237316195423570985008687907853269984665640564039457584007913129639935";
-    // 2 ** 255
-    const MID_DEC_STR: &str =
-        "57896044618658097711785492504343953926634992332820282019728792003956564819968";
-
+    // all 7 of these test cases are from
+    // https://github.com/ethereum/portal-network-specs/blob/master/state-network.md
```

Comment:
Imo, it would be nicer to just name all the tests.. `case_one` .. `case_seven`, or something similar. And then in a comment just above each test paste the actual case over from the specs